src/scss/tools/_a11y.scss

```diff
@@ -1,15 +1,3 @@
-/// Return either white or black for the text base
-/// depending on the background color
-///
-/// @param {String} $color - the name of the background color being used
-@function _oColorsGetTextBase($color) {
-	@if oColorsColorBrightness($color) < 65% {
-		@return 'white';
-	} @else {
-		@return 'black';
-	}
-}
-
 /// Checks the contrast ratio and returns the WCAG
 /// rating, either AAA, AA, or AA18 meaning text should
 /// be at least 18px
```

Comment:
i can't comment on the function itself, but is the `oColorsGetWCAGRating` function used anywhere? i don't see any uses on github other than in places where `o-colors` has been committed

Reply:
Yeah I'm intending on getting rid of that one :) The proposal keeps `oColorsGetContrastRatio` https://github.com/Financial-Times/o-colors/issues/198